dap-python.el

```diff
@@ -45,8 +45,9 @@ If the port is taken, DAP will try the next port."
   "Populate CONF with the required arguments."
   (let* ((host "localhost")
          (debug-port (dap--find-available-port host dap-python-default-debug-port))
-         (python-executable dap-python-executable))
-    (compile (format "%s -m ptvsd --wait --host %s --port %s %s" python-executable host debug-port buffer-file-name))
+         (python-executable dap-python-executable)
+         (python-args (if (plist-get conf :args) (plist-get conf :args) "")))
```

Comment:
could be `(or (plist-get conf :args) "")`

Reply:
sure, shorter and brighter :)